html/semantics/embedded-content/the-iframe-element/iframe-loading-lazy-history-pushState.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<title>History state change for iframe loading='lazy' before it is loaded: history.pushState</title>
+<iframe data-src="support/blank.htm?foo" src="support/blank.htm?src" loading="lazy"></iframe>
+<script>
+const iframe = document.querySelector('iframe');
+// Should have no effect on lazy-loading
+iframe.contentWindow.history.pushState(null, "", iframe.dataset.src);
+</script>
+<!-- Loading testharness.js here is intentional. -->
```

Comment:
Why?

Reply:
WebKit passes the tests if testharness.js is loaded in `head`, but fails these tests (or [a simple demo without external scripts](https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=12473)).

Reply:
I see. Maybe add a brief comment indicating just that, just for extra clarity / archaeology?

Reply:
Oh I think you already did that? In that case, I'll go ahead and mark this thread resolved unless you were leaving it unresolved for a different reason.